### PR TITLE
UIIN-2311 Support to display authorized indicators

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Added icon links to MARC Authority app for controlled Contributors, Series, Uniform titles and Subjects

## Screenshots

https://user-images.githubusercontent.com/19309423/217798696-20bda691-d50d-4a22-97a9-3484a4d81690.mp4

## Issues
[UIIN-2311](https://issues.folio.org/browse/UIIN-2311)

### CODE REVIEW

**Review by Dmytro-Melnyshyn — APPROVED** [review]

File: src/Instance/InstanceDetails/InstanceTitleData/AlternativeTitlesList.js [hunk 1]
```diff
@@ -21,18 +21,30 @@ const getColumnMapping = intl => ({
...
-const formatter = {
-  type: item => item.type || noValue,
-  title: item => item.alternativeTitle || noValue,
-};
 const columnWidths = {
   type: '25%',
   title: '75%',
 };
 
+const getTitleItem = (item, segment, source) => {
```
Comment (mkuklis) [inline src/Instance/InstanceDetails/InstanceTitleData/AlternativeTitlesList.js:29]: @BogdanDenis this looks exactly the same as in [InstanceSubjectView.js](https://github.com/folio-org/ui-inventory/pull/1977/files#diff-b4e449d19d4ce4ec1cea18f63bee488722446eb63a3cb5c1e66f91f13a869fbd)

Perhaps we could extract it so it's reused in both places?

File: src/Instance/InstanceDetails/InstanceTitleData/TitleSeriesStatements.js [hunk 2]
```diff
@@ -9,34 +9,47 @@ import {
...
 const noValue = <NoValue />;
 
 const visibleColumns = ['statement'];
 const getColumnMapping = intl => ({
   statement: intl.formatMessage({ id: 'ui-inventory.seriesStatement' }),
 });
-const formatter = {
-  statement: item => item.statement || noValue,
+
+const getStatementItem = (item, segment, source) => {
```
Comment (mkuklis) [inline src/Instance/InstanceDetails/InstanceTitleData/TitleSeriesStatements.js:22]: @BogdanDenis this is almost the same too.

**Review by mkuklis — APPROVED** [review]
@BogdanDenis this looks good. It would be nice to extract that one function which returns `MarcAuthorityLink` so it can be reused.

File: src/Instance/InstanceDetails/InstanceTitleData/AlternativeTitlesList.js (hunk 1 above)
Comment (BogdanDenis) [reply src/Instance/InstanceDetails/InstanceTitleData/AlternativeTitlesList.js]: That's a good point, I've moved this logic to `<ControllableDetail>` component

File: src/components/InstancesList/InstancesList.js [hunk 3]
```diff
@@ -993,7 +993,7 @@ class InstancesList extends React.Component {
             columnWidths={{
               callNumber: '15%',
               subject: '50%',
-              contributor: '50%',
+              contributor: '200px',
```
Comment (Dmytro-Melnyshyn) [inline src/components/InstancesList/InstancesList.js:996]: Do we need this?

Comment (Dmytro-Melnyshyn) [inline src/components/InstancesList/InstancesList.js:996]: Do we need this?

**Review by Dmytro-Melnyshyn — CHANGES_REQUESTED** [review]

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Pull request merged**